rosdep/base.yaml

```diff
@@ -3051,6 +3051,16 @@ libnl-dev:
   fedora: [libnl-devel]
   gentoo: [dev-libs/libnl]
   ubuntu: [libnl-dev]
+libnlopt-cxx-dev:
+  debian:
+    '*': [libnlopt-cxx-dev]
```

Comment:
@mgruhler is the package available in all distros but ` stretch`?

Reply:
it is in `buster (stable)`, `bullseye (testing)` and `sid (unstable)`. It's also available in `stretch-backports`. But obviously, it is not available in the older ones, i.e. the ones before stretch.

Reply:
Exactly. Perhaps we should be specific if not too verbose.

Reply:
so what do you suggest? should I rather add it specifically for the distros it DOES exist in? i.e.:

```yaml
debian:
  buster: [liblnopt-cxx-dev]
  bullseye: [liblnopt-cxx-dev]
```

Reply:
Do we need to exclude the other ones? i.e. `'*': null`?
drawback is also that we'd have to add them manually for each new distro to come that does contain it.

In the end, I'm fine with whatever... :-)

Reply:
> Do we need to exclude the other ones? i.e. `'*': null`?

Not existing mapping are equivalent to `null`. You only need to set explicit distros to `null` when using the wildcard `*` mapping to some value.

> drawback is also that we'd have to add them manually for each new distro to come that does contain it.

We try to avoid this wherever possible since it is a continuous maintenance burden and likely doesn't happen all the time.

> there are other keys where it is done similar to what I did here, basically exclude the current LTS versions that don't have this package and add for all others to come...

:+1:

Reply:
Hmm, yeah, that's a good point. Alright, disregard my comment.